Formula/vcsh.rb

```diff
@@ -1,19 +1,22 @@
 class Vcsh < Formula
   desc "Config manager based on git"
   homepage "https://github.com/RichiH/vcsh"
-  url "https://github.com/RichiH/vcsh/archive/refs/tags/v1.20190621-4.tar.gz"
-  version "1.20190621"
-  sha256 "178ddf6f7bba15bcc295a08247070665e5b799af64753e21c7fac68f72296ca8"
+  url "https://github.com/RichiH/vcsh/releases/download/v2.0.2/vcsh-2.0.2.tar.xz"
+  sha256 "3ffc0bbb43c76620c8234c98f4ae94d0a99d24bb240497aab730979a8d23ad61"
   license "GPL-2.0"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "d6adf58cb690fa63602d8056d347bf55dde6ebd090c6f82d1bbbbc78b137d93b"
   end
 
   def install
-    bin.install "vcsh"
-    man1.install "vcsh.1"
-    zsh_completion.install "_vcsh"
+    system "./configure", "--with-zsh-completion-dir=#{zsh_completion}",
+                          "--with-bash-completion-dir=#{bash_completion}",
+                          *std_configure_args
+    system "make", "install"
+
+    # Remove references to git shim
+    inreplace bin/"vcsh", %r{#{HOMEBREW_SHIMS_PATH}/[^/]+/super/git}o, "git"
```

Comment:
Thanks a lot for the help with this @carlocab. I do have a question about this line though. Are we doing something wrong upstream detecting the path to `git` at configure time? On is this to correct for an idiosyncrasy in Homebrew where the compile time path is different than what should be the run time path?

Either way I think there is a better way to do this. If the configure time detection (that uses autotools built in program detection) is not turning up the correct answer, you can inform it manually with `./configure GIT=/path/to/git` or even `./configure GIT=git` if using relative paths at runtime is appropriate for the platform. This goes for other tools that are detected as well. That's what bothers me here, if Git is not being detected correctly are other tools?

Also I'm curious why there is no `depends_on "git"` or `uses_from_macos "git"` or similar. Shouldn't there be some sort of dependency declared for this to work reliably?

Reply:
> On is this to correct for an idiosyncrasy in Homebrew where the compile time path is different than what should be the run time path?

Yes, this, at least in the case of Git, and possibly any absolute path on Linux. Is there any danger in querying `PATH` at runtime instead of hardcoding absolute paths into `vcsh`? It seems to me that the former is more flexible, but perhaps there's a reason you're doing this I don't see.

> That's what bothers me here, if Git is not being detected correctly are other tools?

The audit should catch stuff that's definitely wrong, though I'm a bit wary of hardcoding `/bin/sed` and `/bin/grep` as the paths to search for these on Linux at runtime.

> Also I'm curious why there is no `depends_on "git"` or `uses_from_macos "git"` or similar. Shouldn't there be some sort of dependency declared for this to work reliably?

Homebrew itself won't work without Git, so any system with Homebrew installed will also have a system Git installation. Doing `depends_on "git"` or `uses_from_macos "git"` means we want to use the `git` formula (and the latter only on Linux); usually because we need a new enough version of Git for things to work. Is that the case here?